packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -65,23 +68,26 @@ export default class RunPanelStore {
                 'Account switching is disabled while your bot is running. Please stop your bot before switching accounts.'
             )
         );
-
-        this.is_running = true;
-        ui.setPromptHandler(true, route_prompt_dialog.shouldNavigateAfterPrompt);
-        this.toggleDrawer(true);
-        this.run_id = `run-${Date.now()}`;
-
-        contract_card.clear();
-        this.setContractStage(contract_stages.STARTING);
-        this.dbot.runBot();
-    };
+        runInAction(() => {
+            this.is_running = true;
+            ui.setPromptHandler(true, route_prompt_dialog.shouldNavigateAfterPrompt);
+            this.toggleDrawer(true);
+            this.run_id = `run-${Date.now()}`;
+
+            contract_card.clear();
+            this.setContractStage(contract_stages.STARTING);
+            this.dbot.runBot();
+        });
```

Comment:
No need to wrap this in `runInAction`, the function the code lives in is already decorated with `action.bound`.